src/main/java/com/keviiweb/keips/ExcelReader.java

```diff
@@ -10,58 +10,103 @@
  *
  * Code adapted from: https://www.mkyong.com/java/apache-poi-reading-and-writing-excel-file-in-java/
  * and https://poi.apache.org/components/spreadsheet/quick-guide.html
  */
-public class ExcelReader
-{
-    public static void main( String[] args ) {
-        if (args.length < 2) {
-            System.out.println("Usage: run java ExcelReader <excel file in .xlsx format> <password>");
-            return;
-        }
 
-        final String FILENAME = args[0];
-        final String password = args[1];
-        System.out.println( "Reading from " + FILENAME + " with password: " + password);
+public class ExcelReader {
+	
+	private FileInputStream excelFile;
+	private Workbook workbook;
+	private DataFormatter formatter;
+	
+    public static void main( String[] args ) {
+    	
+    	ExcelReader newReader = new ExcelReader();
+    	newReader.run();
+    	
+	}
+	
+	private void run() {
+		Scanner sc = new Scanner(System.in);
 
-        try {
-            FileInputStream excelFile = new FileInputStream(new File(FILENAME));
-            Workbook workbook = WorkbookFactory.create(excelFile, password);
-            DataFormatter formatter = new DataFormatter();
-            Sheet mainSheet = workbook.getSheetAt(0);
-            Sheet bonusSheet = workbook.getSheetAt(1);
+		while (true) {
+			String fileName = sc.nextLine();
+			System.out.println("User input: " + fileName);
 
-            Iterator<Row> iterator = mainSheet.iterator();
-            iterator.next();
+			//terminate if user input is -1
+			if (fileName.equals("-1")) {
+				System.out.println("Terminating program ...");
+				return;
+			}
 
-            while (iterator.hasNext()) {
-                Row currentRow = iterator.next();
-                Iterator<Cell> cellIterator = currentRow.iterator();
-                List<String> nameRow = new ArrayList<>();
+			if (!parseFile(fileName)) {
+				System.out.println("Error parsing file.");
+			} else {
+				System.out.println("Successfully parsed file.");
+			}
+		}
+	}
+	
+	//takes in as input the filename and parses the file
+	private boolean parseFile(String fileName) {
+    	//set up the excelfile
+    	try {
+    		excelFile = new FileInputStream(new File(fileName));
+    		workbook = WorkbookFactory.create(excelFile);
+    		formatter = new DataFormatter();
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+			return false;
+		} catch (IOException e) {
+			e.printStackTrace();
+			return false;
+		} catch (InvalidFormatException e) {
+			e.printStackTrace();
+			return false;
+		}
 
-                while (cellIterator.hasNext()) {
-                    Cell currentCell = cellIterator.next();
-                    String cellText = formatter.formatCellValue(currentCell);
-                    nameRow.add(cellText);
-                }
-                processName(nameRow);
-            }
+		Iterator<Sheet> sheetiterator = workbook.sheetIterator();
+		
+    	int sheetCount = 0;
+    	while (sheetiterator.hasNext()) {
```

Comment:
For now, I think we just take the first sheet in as we're not sure how each file will be structured.